site/content/blog/2020-05-04-pritunl.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "From OpenVPN to Pritunl VPN: The transition"
+slug: pritunl
+date: 2020-05-04T12:00:00-04:00
```

Comment:
Need to update the date to latest before merge

Reply:
you're right Joram! will update it before merging